### DISCUSSION TITLE
Slow startup time

### DISCUSSION THREAD

**Discussion by tealsnow** [Aylur/ags#607] [Q&A] [answered]
Is there any way for me to speed up the startup time. It takes a whole minute from login to when the shell shows up. Perhaps some debugging methods or tips? Or any other ideas?

**Comment by Jas-SinghFSU** [comment]
This is extremely unusual. Do you have a blocking process?

**Comment by Aylur** [comment, marked as answer]
Iirc, this is an issue with gtk apps when here is no desktop-portal running
